src/components/CardList/index.js

```diff
@@ -14,21 +14,20 @@
  */
```

Comment:
@connor-baer You sure this one looks correct? It says GlobalStyles -> CardList

Reply:
I saw that, too. It's just `git` being confused.